examples/vanilla/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@compassion-gds/examples-vanilla-webpack",
+  "version": "0.2.0-alpha.0",
+  "description": "An example of @compassion-gds/css bundled with webpack",
+  "source": "src/index.html",
+  "scripts": {
+    "start": "webpack serve --open",
+    "bundle": "webpack --mode=production"
+  },
+  "private": true,
+  "dependencies": {
+    "@compassion-gds/css": "file:../../packages/css"
```

Comment:
BTW this is because `examples` are not part of the workspace structure - so that their dependencies aren't added / bootstrapped to the monorepo